src/index.html

```diff
@@ -5,8 +5,361 @@
+            <div class="card__code">Product code: 195434</div>
+            <div class="card__review-wrapper">
+              <div class="card__stars-container card__stars--4 stars">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <div class="card__review">
+                <p>Reviews: 5</p> 
+              </div>
+            </div>
+            <div class="card__price price">
+              <div class="price__label">
+                <p>Price:</p>
+              </div>
+              <div class="price__number">
+                <p>$2,199</p>
+              </div>
+            </div>
+            <div>
+              <a href="#" data-qa="card-hover">
+                <button type="button"  class="card__button">
+                  BUY
+                </button>
+              </a>
+            </div>
+          </div>
+        </div>
+        <div class="container__card card">
+          <img 
+            class="card__image" 
+            src="./images/imac.jpeg" 
+            alt="imac"
+          >
+          <div class="card__information">
+            <div class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
+            <div class="card__code">Product code: 195434</div>
+            <div class="card__review-wrapper">
+              <div class="card__stars-container card__stars--4 stars">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <div class="card__review">
+                <p>Reviews: 5</p> 
+              </div>
+            </div>
+            <div class="card__price price">
+              <div class="price__label">
+                <p>Price:</p>
+              </div>
+              <div class="price__number">
+                <p>$2,199</p>
+              </div>
+            </div>
+            <div>
+              <a href="#" data-qa="card-hover">
+                <button type="button"  class="card__button">
+                  BUY
+                </button>
+              </a>
+            </div>
+          </div>
+        </div>
+        <div class="container__card card">
+          <img 
+            class="card__image" 
+            src="./images/imac.jpeg" 
+            alt="imac"
+          >
+          <div class="card__information">
+            <div class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
+            <div class="card__code">Product code: 195434</div>
+            <div class="card__review-wrapper">
+              <div class="card__stars-container card__stars--4 stars">
```

Comment:
```suggestion
              <div class="card__stars-container card__stars stars stars--4">
```
модифікатор краще додавати саме до блоку